packages/base-zone/src/prepare-revocable.js

```diff
@@ -76,15 +76,25 @@ import { wrapperMethods } from './prepare-attenuator.js';
  * @param {PropertyKey[]} uMethodNames
  *   The method names of the underlying exo class that should be represented
  *   by transparently-forwarding methods of the revocable caretaker.
- * @param {(amp: Amplify<any>) => void} [receiveAmplifier]
  * @param {RevocableKitOptions<U>} [options]
+ *
+ * @param {(amp: Amplify<any>) => void} [receiveAmplifier]
+ *   This parameter is implemented by the `FarClassOptions['receiveAmplifier']`
+ *   option to `exoClassKit`.
+ *
+ *   If provided by the caller, it will be called back exactly once to
+ *   expose the `Amplify` function which, when called on any of a Kit
+ *   instance's facets, will return the complete `facets` object.
+ *
+ *   `receiveAmplifier` is specified by `prepareRevocableMakerKit` to allow
+ *   `revoke(myKit.revocable)` to obtain and call `myKit.revoker.revoke()`.
  */
 export const prepareInternalRevocableMakerKit = (
   zone,
   uKindName,
   uMethodNames,
-  receiveAmplifier,
   options = {},
+  receiveAmplifier = undefined,
```

Comment:
changing the parameter order looks like a breaking change. is there something I'm not seeing?

Reply:
oh... this is a new function?

Reply:
It is a new function, but it isn't a change to parameter order. Rather it is adding an additional optional parameter. However, when the last existing parameter is an optional options bag, it is usually better to put make new optional parameters as named parameters within that options bag.